swarm/api/storage.go

```diff
@@ -1,70 +0,0 @@
-// Copyright 2016 The go-ethereum Authors
```

Comment:
Given swarm is pre 1.0 (I believe 0.2?) and this IPC API is pretty much undocumented, it is totally acceptable to make breaking changes or remove it entirely (this is [encouraged by semver](http://semver.org/#doesnt-this-discourage-rapid-development-and-fast-iteration)). IPC clients can easily switch to the improved HTTP API, and library users can just call `Get` and `Put` on the `api.API` object instead.

At this stage we should really not be trying to incur technical debt, rather iterating until we reach an API we consider stable enough to call 1.0.